posthog/clickhouse/migrations/0098_sessions_v2_irclid_kx.py

```diff
@@ -0,0 +1,23 @@
+from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
+from posthog.models.raw_sessions.migrations import (
+    WRITABLE_RAW_SESSIONS_ADD_VITALS_LCP_COLUMN_SQL,
+    DISTRIBUTED_RAW_SESSIONS_ADD_VITALS_LCP_COLUMN_SQL,
+    BASE_RAW_SESSIONS_ADD_VITALS_LCP_COLUMN_SQL,
+)
+from posthog.models.raw_sessions.sql import (
+    DROP_RAW_SESSION_MATERIALIZED_VIEW_SQL,
+    RAW_SESSIONS_TABLE_MV_SQL,
+    RAW_SESSIONS_CREATE_OR_REPLACE_VIEW_SQL,
+)
+
+operations = [
+    # drop the mv, so we are no longer receiving events from the sessions table
+    run_sql_with_exceptions(DROP_RAW_SESSION_MATERIALIZED_VIEW_SQL()),
+    # now we can alter the target tables
+    run_sql_with_exceptions(WRITABLE_RAW_SESSIONS_ADD_VITALS_LCP_COLUMN_SQL()),
+    run_sql_with_exceptions(DISTRIBUTED_RAW_SESSIONS_ADD_VITALS_LCP_COLUMN_SQL()),
+    run_sql_with_exceptions(BASE_RAW_SESSIONS_ADD_VITALS_LCP_COLUMN_SQL()),
```

Comment:
Unrelated: We don't need this LCP column anymore - I believe. What would it take to remove it?

1. First PR removing it from the `sessions` Django table
2. Second PR removing it from the DB?

Is that all?

Reply:
Step 1: see if anyone is using it in any queries ;)

The following steps, yes that sounds right

Reply:
Ok, people are using it, plan is over 😢

My metabase query for future reference: `SELECT * FROM system.query_log WHERE type = 'QueryStart' AND query LIKE '%vitals_lcp%' AND log_comment LIKE '%vitals_lcp%' AND event_time >= makeDate(2025, 01, 01) LIMIT 5;`